tests/lambda_functions/vetext_incoming_forwarder_lambda/test_vetext_incoming_forwarder_lambda.py

```diff
@@ -167,9 +166,50 @@ def test_failed_getenv_vetext_api_endpoint_path(mocker, event):
 @pytest.mark.parametrize('event', [(albInvokedWithoutAddOn), (albInvokeWithAddOn), (sqsInvokedWithAddOn)])
 @pytest.mark.usefixtures('os_environ', 'missing_ssm_path_env_param')
 def test_failed_getenv_vetext_api_auth_ssm_path(mocker, event):
-    sqs_mock = mocker.patch(f'{LAMBDA_MODULE}.push_to_sqs')
+    sqs_mock = mocker.patch(f'{LAMBDA_MODULE}.push_to_retry_sqs')
     mocker.patch(f'{LAMBDA_MODULE}.read_from_ssm', return_value="ssm")    
     response = vetext_incoming_forwarder_lambda_handler(event, None)
 
     assert response['statusCode'] == 200
-    sqs_mock.assert_called_once()
\ No newline at end of file
+    sqs_mock.assert_called_once()
+
+@pytest.mark.usefixtures('os_environ', 'all_path_env_param_set')
+def test_unexpected_event_received(mocker):
+    sqs_dead_letter_mock = mocker.patch(f'{LAMBDA_MODULE}.push_to_dead_letter_sqs')   
+    event = {}
+    response = vetext_incoming_forwarder_lambda_handler(event, None)
+
+    assert response['statusCode'] == 400
+    sqs_dead_letter_mock.assert_called_once()
+
+@pytest.mark.parametrize('event', [(sqsInvokedWithAddOn)])
+@pytest.mark.usefixtures('os_environ', 'all_path_env_param_set')
+def test_failed_getenv_vetext_api_auth_ssm_path(mocker, event):
+    sqs_dead_letter_mock = mocker.patch(f'{LAMBDA_MODULE}.push_to_dead_letter_sqs')   
+
+    mocker.patch("json.loads", side_effect=Exception)
+    response = vetext_incoming_forwarder_lambda_handler(event, None)
+
+    assert response['statusCode'] == 200
+    sqs_dead_letter_mock.assert_called_once()
+
+@pytest.mark.parametrize('event', [(albInvokedWithoutAddOn)])
+@pytest.mark.usefixtures('os_environ', 'all_path_env_param_set')
+def test_loading_message_from_alb_fails(mocker, event):
+    sqs_dead_letter_mock = mocker.patch(f'{LAMBDA_MODULE}.push_to_dead_letter_sqs')   
+
+    mocker.patch("base64.b64decode", side_effect=Exception)
+    response = vetext_incoming_forwarder_lambda_handler(event, None)
+
+    assert response['statusCode'] == 200
+    sqs_dead_letter_mock.assert_called_once()
+
+@pytest.mark.parametrize('event', [(albInvokedWithoutAddOn)])
```

Comment:
Why parametrize is single event?  Ditto for other occurrences.  You don't need the parenthesis because this is a single parameter.  "[(albInvokeWithoutAddOn)}" evaluates to a list with a single element that has the dictionary type.  If you wanted a 1-tuple element, you would write "[(albInvokeWithoutAddOn,)]" (note the comma).